docs/guide/static-module-assets.md

```diff
@@ -0,0 +1,45 @@
+# Using static module assets
+
+While developers can place static files in the top-level `/public` folder of the project, which allows a file like `/public/images/bg.png` to be accessed via the URL `https://examplesite.com/images/bg.png`, this isn't a good solution for some situations.
```

Comment:
For the URL let's use `example.com` since that's the pattern used elsewhere and doesn't ask the reader to parse the two words while reading.

Reply:
Bumping this one.